src/System.Threading.Thread/ref/project.json

```diff
@@ -1,6 +1,9 @@
 {
   "dependencies": {
-    "System.Runtime": "4.4.0-beta-24625-01"
+    "System.Runtime": "4.4.0-beta-24625-01",
+    "System.Runtime.Extensions": "4.4.0-beta-24625-01",
+    "System.Security.Principal": "4.4.0-beta-24625-01",
+    "System.Threading": "4.4.0-beta-24625-01"
```

Comment:
> "System.Threading": "4.4.0-beta-24625-01" [](start = 3, length = 42)

@ericstj I believe this won't actually introduce a dependency cycle because if I'm correct, System.Threading depends on System.Threading.Tasks and not on Threading.Thread, but I'm just curious if we have a way of catching dependency cycles with our CI, or if the only way to catch them is when we actually try to update the corefx dependencies that contain the cycle?